### ISSUE TITLE
Update file description for lake_temp_preds_etcetc

### ISSUE THREAD

**Issue by lindsayplatt** [USGS-R/lake-temp-lstm-static-data-release#50]
From Sam:

> - Define “DSG” in entity definition.  
> - The description of how the files are organized is hard to follow. Maybe after the sentence “The {STATE SUBSET OR GCM TYPE} field (if present) further groups the NetCDFs in order for the zip files to remain under 10 GB.” you could have a bullet point for each high level zip and describe what’s in it?

**Comment by jzwart** [comment]
low priority 
Discrete Sampling Geometries (DSG) is sufficient IMO (in my opinion :) ) 

I feel like it is described pretty well already with: 
> The "EALSTM_NLDAS" predictions were grouped into subsets alphabetically by state: the "AR-MN" file contains NetCDFs for AR, IA, IL, IN, KS, KY, LA, MI, and MN, while the "MO-WY" file contains NetCDFs for MO, MS, MT, ND, NE, OH, OK, SD, TN, TX, WI, and WY. The "GLM_NLDAS" predictions fit into a single zip file with NetCDFs for 11 states: AR, IA, IL, IN, MI, MN, MO, ND, OH, SD, and WI. Lastly, the "GLM_GCM" predictions had to be grouped into separate CSVs based on the 6 different climate models used (ACCESS, CNRM, GFDL, IPSL, MIROC5, and MRI), each of those zips contains one NetCDF per state (same 11 states as "GLM_NLDAS").
